content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -28,7 +28,7 @@ resources instead of a single large Endpoints resource.
 
 ## Enabling EndpointSlices
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+{{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
 {{< note >}}
```

Comment:
Is this note still accurate? Maybe it merits a slight rewording.

Reply:
In fact, is the whole task page still needed? We could replace it with a redirect to the EndpointSlice concept page.

Reply:
That is a good point. There are still a few flags that have not been graduated yet (`EndpointSliceProxying`, `EndpointSliceTerminating`, `WindowsEndpointSliceProxying`). What would be the best place to add the information related to those feature gates? I am not sure if it should be moved to the concepts page or if there is a better home for it.

Reply:
I would be happy to see `content/en/docs/tasks/administer-cluster/enabling-endpointslices.md` removed so long as we add a redirect to the concept page about EndpointSlice.

It's OK not to have a task page for feature flags that are relatively niche. EndpointSlice got a task page for enabling it because it's relevant to a lot of clusters and use cases.

Reply:
In that case, I have removed this page, and added it to the redirects.